docs/technical/installation_controller.md

```diff
@@ -0,0 +1,38 @@
+# Installation Controller
+
+The reconciliation flow of the installation controller is described in detail [here](./reconciliation_flow.md).
+It executes one of the following main operations:
+
```

Comment:
I dont find the documentation of all the phases in this folder. it would be useful to have it here (direct or referenced), since the phases are used in the following flows and explanations.

Reply:
I added a paragraph about the phases and what the "combined phase" is.